pkg/admission/server.go

```diff
@@ -43,6 +43,11 @@ var (
 		Version:  v1alpha1.SchemeGroupVersion.Version,
 		Resource: "httproutes",
 	}
+	gatewayGVR = meta.GroupVersionResource{
+		Group:    v1alpha1.SchemeGroupVersion.Group,
+		Version:  v1alpha1.SchemeGroupVersion.Version,
```

Comment:
#651 was merged in. Can we instead use v1alpha2?

This leads to a bigger question, do we want to implement validation for the latest version or add it for v1alpha2 as well?
I expect the transition from v1alpha1 to 2 will be slow but then do we continue to support alpha1 or give users more reasons to adopt alpha2 sooner.

cc @robscott @danehans

Reply:
I think we should plan on supporting at least the current and trailing version of the API. That would mean v1alpha1 and v1alpha2 now, eventually transitioning to v1beta1 and v1alpha2 in the future.

Reply:
I just rebased & added support for both versions.